.ci/azure/mac.yml

```diff
@@ -17,7 +16,6 @@ pr:
     - releases/*
   paths:
     exclude: 
-    - docs/
```

Comment:
CC @ilyachur

Reply:
this is only for mac build test suggested by @ilyachur , will be revert after validate.

Reply:
@zhaixuejun1993 You don't need to do it. Before you did it only because you changed snippets (for snippets you also shouldn't do it anymore, I added additional steps to check the snippets), but now you are changing benchmart_app, you don't touch the documentation.

Reply:
> 

Get it. Thanks! Will revert the change